### PR TITLE
Fix array size in bom_bowl_man

### PR DESCRIPTION
**Pull request by louist103**
The array is 1 indexed with the 0th element being the red shirt guy.

### CODE REVIEW

**Review by engineer124 — APPROVED** [review]

File: src/overlays/actors/ovl_En_Bom_Bowl_Man/z_en_bom_bowl_man.h [hunk 1]
```diff
@@ -49,8 +49,7 @@ typedef struct EnBomBowlMan {
     /* 0x2D2 */ s16 csId2;
     /* 0x2D4 */ s16 unk_2D4;
     /* 0x2D6 */ s16 csId3;
-    /* 0x2D8 */ struct EnBomBowlMan* unk_2D8[5];
-    /* 0x2EC */ UNK_TYPE1 unk2EC[0x4];
+    /* 0x2D8 */ struct EnBomBowlMan* unk_2D8[6];
```
Comment (engineer124) [inline src/overlays/actors/ovl_En_Bom_Bowl_Man/z_en_bom_bowl_man.h:52]:
```suggestion
    /* 0x2D8 */ struct EnBomBowlMan* unk_2D8[6]; // The array is 1 indexed with the 0th element being the red shirt bomber
```
Could add the comment here too keep the knowledge as well

**Review by hensldm — APPROVED** [review]

**Review by engineer124 — APPROVED** [review]

File: src/overlays/actors/ovl_En_Bom_Bowl_Man/z_en_bom_bowl_man.c [hunk 2]
```diff
@@ -656,8 +656,7 @@ void func_809C5BF4(EnBomBowlMan* this, PlayState* play) {
                 this->actionFunc = func_809C4DA4;
             } else {
                 s32 idx = D_809C6100 - 1;
-
-                Camera_SetTargetActor(subCam, &this->unk_2D8[1 + idx]->actor);
+                Camera_SetTargetActor(subCam, &this->unk_2DC[idx]->actor);
```
Comment (engineer124) [inline src/overlays/actors/ovl_En_Bom_Bowl_Man/z_en_bom_bowl_man.c:659]: Somehow lost the new line after the variable declaration, could you restore it?

**Comment by hensldm** [comment]
Looks like this could use a format.

**Comment by starlightgemini214** [comment]
I remember playing this when i was younger and i found this weird looking NPC with bug eyes and he gave me like 50 rupees for finding him. His name was like squimble or swimble or something like that and I have never seen him mentioned anywhere. I tried finding him on my 3DS but never could. 

Does anyone else remember this or am I going insane??

**Comment by starlightgemini214** [comment]
I understand that the butler is supposed to be at the shrine to start a game so I can get the mask, but I only had 10 seconds left before the end of day 3 when finally got the princess back to the king, so I had to warp to an owl, save and reset back to day 1. Will I need to redo the dungeon or something to that effect? I am at the Deku Palace now and the NPC is (obviously) behaving as they did before the completion of the dungeon. Please help! :(

**Comment by ginomeelectra202** [comment]
The title explanes it......

**Comment by altoidmerope294** [comment]
I'm not sure about this, but is there a use for the ladder behind the Milk Village place in Romani Ranch? If so, can you please tell me?

**Comment by ginomeelectra202** [comment]
.yna em lles t'now dna meht esu ot woh wonk t'nod i taht syas eh kniL ukeD sa emos yub ot yrt i revenehW

**Comment by ginomeelectra202** [comment]
?esuoh naeco eht ni sredips owt eht teg i od woH.ti hcaer t'nac i dna pam eht no H ro ,G,F yas t'nseod tI

**Comment by hirottau302** [comment]
It has been a while since I've played, so I'll try to remember. I'm at the snowy mountain part, I've already gotten the Lens of Truth, and the last thing I've done was go to that one Goron on top of the cliff that's dead (I think he's the father of the baby) and he says, "Heal my sorrows." That's the only phrase I remember because it was in red. What do I do now? I've tried going to the baby and talking to some people but nothing happens. If you can, try to give me a small hint on what to do.

**Comment by altoidmerope294** [comment]
So when i brought back the princess it let me do the deku butler race which i did on time but instead gave me a heartpiece instead of the mask of scents and now i still cant get the mask instead just gives me 20 ruppees so what do i do now and can i still the mask by any other way?

**Review by tom-overton — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**